queryfilter/base.py

```diff
@@ -21,3 +35,22 @@ def on_sqlalchemy_query(self, query):
     @abc.abstractmethod
     def on_dicts(self, dicts):
         pass
+
+    def false_with_drop_none_else_raise(self, field_name):
+        if self.options["drop_none"]:
+            return False
+        raise FilterOnNoneValueError(field_name)
+
+
+class BaseDictFilter(object):
```

Comment:
Maybe name with xxxBase or xxxMixin to replace Filter? since without logic related with filter here 🤔